vitrina/uapi/views/views.py

```diff
@@ -106,20 +106,26 @@ def get_serializer_class(self) -> Serializer:
 
     @transaction.atomic
     def create(self, request: Request, *args: Any, **kwargs: Any) -> Response:
+        organization = self.request.organization
         serializer_context = self.get_serializer_context()
         serializer = self.get_serializer(
             data=request.data,
             context={
                 **serializer_context,
-                "organization": self.request.organization,
+                "organization": organization,
             }
         )
         serializer.is_valid(raise_exception=True)
         with create_revision():
-            instance = serializer.save(
-                subclass=self.dcat_resource_subclass,
-                access_rights=Dataset.NON_PUBLIC
-            )
+            validated_data = serializer.validated_data
```

Comment:
Would be a better practice.
```suggestion
            validated_data = serializer.validated_data.copy()
```

Reply:
Resolved on a call